### DISCUSSION TITLE
Until when will v19 be supported?

### DISCUSSION THREAD

**Discussion by fballiano** [OpenMage/magento-lts#2529] [Questions & Answers]
Somewhere I've read 5 years but I can't find that page anymore.

Should we write something in the README?

**Comment by Flyingmana** [comment]
It was for example here https://github.com/OpenMage/OpenMage.github.io/blob/master/_posts/2019-12-09-release-19-4-0.md

and also in this RFC in Alternative 2 https://github.com/OpenMage/rfcs/blob/main/draft/0000-release-schedule.md

**Reply by fballiano** [reply]
ok, the idea was 5 years (which is a very, very long time) but nothing was ever set in stone right?

**Reply by Flyingmana** [reply]
one could argue I did set it into stone by writing a blogpost for it 😅
But I still say, RFCs overrule everything

**Reply by fballiano** [reply]
fine, but I wouldn't consider https://github.com/OpenMage/rfcs/blob/main/draft/0000-release-schedule.md since it doesn't seem to be finished.

**Reply by Flyingmana** [reply]
yes, but a published blog post describing it is strong enough to not be simple ignored. Its basically a marketing promise, which needs a strong weight to be overruled.

besides, half of the time is already over and we have no decided on alternative yet.

**Reply by fballiano** [reply]
I said fine, meaning that I agree that the blog post counts.

**Comment by sreichel** [comment]
https://docs.openmage.org/users/versioning/maintained/
